### PR TITLE
Fix Project Screen edit functionality

### PR DESCRIPTION
**Pull request by harshaanNihal**
## Notion card
https://www.notion.so/saeloun/Add-Project-is-not-working-on-Production-0bb9e1172faf4b8290f45855de30804c

## Summary

- Fix edit functionality and Code refactor

## Preview

Added Edit/create/delete preview
https://www.loom.com/share/18b050179dc348a299a31f49427413ef

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by supriya3105** [comment]
@onkar-saeloun @akhilgkrishnan Please review this PR on priority as this a production bug. @apoorv1316 Please test this and merge once approved.

**Review by akhilgkrishnan — APPROVED** [review]
LGTM

File: app/javascript/src/components/Projects/Modals/AddEditProject.tsx [hunk 1]
```diff
@@ -18,6 +18,9 @@ const AddEditProject = ({
     || (editProjectData && editProjectData["id"])
     || Number(window.location.pathname.split("/").at(-1));
 
+  const isEdit = !!projectId;
```
Comment (onkar-saeloun) [inline app/javascript/src/components/Projects/Modals/AddEditProject.tsx:21]: why we are using !!projectId? can't we just use projectId?

**Review by onkar-saeloun — APPROVED** [review]
Just added a question LGTM apart from that.

File: app/javascript/src/components/Projects/Modals/AddEditProject.tsx (hunk 1 above)
Comment (harshaanNihal) [reply app/javascript/src/components/Projects/Modals/AddEditProject.tsx:21]: Trying to keep the value isEdit a boolean.

**Pull request merged**